### PR TITLE
Add `core.token.updateTransferSteps` method

### PR DESCRIPTION
**Pull request by llunaCreixent**
This new method calls the `circles-api` endpoint `api/transfers/update` implemented in this PR: https://github.com/CirclesUBI/circles-api/pull/119.

Moreover, the new test simulates a data error by editing the `edges.json` file directly in the `circles-api` container of the test environment. The test checks that a transfer that would fail due to invalid transfer steps, will succeed after calling the new `core.token.updateTransferSteps` method.

For testing this branch please use docker image `v1.6.0-prerelease-bb2bb96`.

Tasks before merging:
- [x] Undo [this commit](https://github.com/CirclesUBI/circles-core/pull/116/commits/b1e69436feced38df12ff2d28324d126f51c3b0b)

### CODE REVIEW

**Comment by llunaCreixent** [comment]
Tests pass with configuration in `test-transfers-update` branch of `circles-docker` repo.

See also tests passing in GH Action: https://github.com/CirclesUBI/circles-core/actions/runs/2405036156

**Comment by JacqueGM** [comment]
I got this error 

```
 Test suite failed to run

    Cannot find module 'fast-json-stringify' from 'test/token.test.js'

      1 | import { execSync } from 'child_process';
    > 2 | import fastJsonStringify from 'fast-json-stringify';
        | ^
      3 | 
      4 | import { getTokenContract } from '~/common/getContracts';
      5 | import getContracts from '~/common/getContracts';

      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:306:11)
      at Object.<anonymous> (test/token.test.js:2:1)
```

**Comment by llunaCreixent** [comment]
> I got this error 
> 
> ```
>  Test suite failed to run
> 
>     Cannot find module 'fast-json-stringify' from 'test/token.test.js'
> 
>       1 | import { execSync } from 'child_process';
>     > 2 | import fastJsonStringify from 'fast-json-stringify';
>         | ^
>       3 | 
>       4 | import { getTokenContract } from '~/common/getContracts';
>       5 | import getContracts from '~/common/getContracts';
> 
>       at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:306:11)
>       at Object.<anonymous> (test/token.test.js:2:1)
> ```

Have you done npm install before running the tests?

**Comment by JacqueGM** [comment]
I did but somehow it keeps failing - but I see pass here so all good. I will approve. I think the issue is locally.

**Comment by JacqueGM** [comment]
ok, I fixed my issue with npm install but running the test in this branch I get this

**Comment by JacqueGM** [comment]
nevermind - my setup was wrong I ran the tests and they passed
`core: 2.10.12`
`api:v1.6.0-prerelease-bb2bb96`
`graph-node:0.26.0`
`relayer: 4.1.11`

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -49,7 +49,7 @@ jobs:
         uses: actions/checkout@v2
         with:
           repository: CirclesUBI/circles-docker.git
-          ref: main
+          ref: test-transfers-update
```
Comment (JacqueGM) [inline .github/workflows/tests.yml:52]: change this line back to main before merging

**Review by JacqueGM — APPROVED** [review]

File: .github/workflows/tests.yml (hunk 1 above)
Comment (llunaCreixent) [reply .github/workflows/tests.yml]: Done :)

**Review by mikozet — APPROVED** [review]

**Comment by llunaCreixent** [comment]
It was expected that the test failed with the last commit: [a27c4b5](https://github.com/CirclesUBI/circles-core/pull/116/commits/a27c4b5cbeb96d48db38aebf95b4c4db66a626a0)

**Pull request merged**